scripts/signedbuildskadi.sh

```diff
@@ -1,274 +1,311 @@
+      NGINX_USER="skadi_$(openssl rand -base64 3)"
+      NGINX_PASSWORD=$(openssl rand -base64 32 |sha256sum | sed 's/ //g')
+      GRAFANA_USER=$NGINX_USER
+      GRAFANA_PASSWORD=$NGINX_PASSWORD
+      SKADI_PASS=$(openssl rand -base64 32 |sha256sum | sed 's/ //g')
+      SKADI_USER_HOME="/home/$SKADI_USER"
+      echo "  Proxy & Grafana Account:" > /opt/skadi_credentials
+      echo "     - Username: $NGINX_USER" >> /opt/skadi_credentials
+      echo "     - Password: $NGINX_PASSWORD" >> /opt/skadi_credentials
+      echo "" >> /opt/skadi_credentials
+      echo "  TimeSketch Account:" >> /opt/skadi_credentials
+      echo "     - Username: $TIMESKETCH_USER" >> /opt/skadi_credentials
+      echo "     - Password: $TIMESKETCH_PASSWORD" >> /opt/skadi_credentials
+  else
+      echo "Using Skadi default username and password of skadi:skadi for OS Account, TimeSketch, Nginx proxy, and Grafana"
+  fi
+}
+
+setup_docker () {
+  # Add Docker gpg key
+  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+
+  # Install Docker
+  sudo apt-get update
+  sudo apt-get install docker-ce -y
+  sudo systemctl enable docker
+
+  # Install Docker-Compose
+  sudo curl -L "https://github.com/docker/compose/releases/download/1.23.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+  sudo chmod +x /usr/local/bin/docker-compose
+  sudo curl -L https://raw.githubusercontent.com/docker/compose/1.23.1/contrib/completion/bash/docker-compose -o /etc/bash_completion.d/docker-compose
+
+  # Add skadi to docker usergroup
+  sudo usermod -aG docker $SKADI_USER
+}
+
+cdqr_cylr_setup () {
+  # Clone skadi and it's submodules
+  sudo git clone --recurse-submodules --branch $install_branch https://github.com/orlikoski/Skadi.git /opt/Skadi
+  sudo chown -R $SKADI_USER:$SKADI_USER /opt/Skadi
+
+  # Copy cdqr script to /usr/local/bin
+  sudo cp /opt/Skadi/scripts/cdqr /usr/local/bin/cdqr
+  sudo chmod +x /usr/local/bin/cdqr
+
+  # Create CyLR directory
+  sudo mkdir /opt/CyLR/
+  sudo chmod 777 /opt/CyLR
+
+  # Installs and Configures CDQR and CyLR
+  sudo -E bash /opt/Skadi/scripts/update.sh
+}
+
+timesketch_configs () {
+  # Write TS and Postgres creds to .env file
+  cd /opt/Skadi/Docker/
+  echo TIMESKETCH_USER=$TIMESKETCH_USER > ./.env
+  echo TIMESKETCH_PASSWORD=$TIMESKETCH_PASSWORD >> ./.env
+  echo POSTGRES_USER=$POSTGRES_USER >> ./.env
+  echo POSTGRES_PASSWORD=$psql_pw >> ./.env
+  echo HEAP_SIZE=1g >> ./.env
+
+  # Configure /etc/hosts file so the host can use same names for each service as the TimeSketch Dockers
+  echo 127.0.0.1       elasticsearch |sudo tee -a /etc/hosts
+  echo 127.0.0.1       postgres |sudo tee -a /etc/hosts
+  echo 127.0.0.1       neo4j |sudo tee -a /etc/hosts
+  echo 127.0.0.1       redis |sudo tee -a /etc/hosts
+
+  # Write TimeSketch config file on host

```

Comment:
This needs to be modified to not put anything on the host and should use the files in the git repo itself (local pathing)